components/Stepper/style.js

```diff
@@ -0,0 +1,157 @@
+import styled from 'styled-components';
+import { BREAKPOINTS } from '../shared';
+import { query } from '../Grid/sub-components/shared';
+
+const MOBILE_CIRCLE_SIZE = 64;
+const DESKTOP_CIRCLE_SIZE = 72;
+const DESKTOP_OVERLAY_WIDTH = 60;
+const DESKTOP_OVERLAY_HEIGHT = 40;
+const MOBILE_OVERLAY_WIDTH = 52;
+const MOBILE_OVERLAY_HEIGHT = 34;
+
+const mediaQueries = query(BREAKPOINTS);
+const desktopBreakpoint = mediaQueries.medium;
+
+const Wrapper = styled.div`
+  display: flex;
+  align-items: center;
+  min-width: 300px;
+
+  .progress {
+    background-image: ${({
+      degrees: { 0: reference, 1: position },
+      theme: { colors: Colors },
+      skin,
+    }) => `
+    ${
+      reference > 0
+        ? `linear-gradient(${reference}deg, ${Colors.neutral[300]} 50%, transparent 50%), 
+      linear-gradient(${position}deg, ${Colors.neutral[300]} 50%, ${Colors[skin][500]} 50%)`
+        : `linear-gradient(${reference}deg, ${Colors[skin][500]} 50%, transparent 50%), 
+      linear-gradient(${position}deg, ${Colors[skin][500]} 50%, ${Colors.neutral[300]} 50%)`
+    }`};
+  }
+`;
+
+const RadialProgressBar = styled.div`
+  border-radius: 50%;
+  display: flex;
+  float: left;
+  font-weight: 700;
+
+  width: ${MOBILE_CIRCLE_SIZE}px;
+  height: ${MOBILE_CIRCLE_SIZE}px;
+
+  ${({ theme: { baseFontSize } }) => `
+    font-size: ${baseFontSize * 0.75}px; 
+  `}
+
+  ${desktopBreakpoint`
+    width: ${DESKTOP_CIRCLE_SIZE}px;
+    height: ${DESKTOP_CIRCLE_SIZE}px;
+
+    ${({ theme: { baseFontSize } }) => `
+    font-size: ${baseFontSize * 0.875}px; 
+    `}
+  `}
+`;
+
+const RadialProgressOverlay = styled.div`
+  ${({
+    theme: {
+      colors: { neutral },
+    },
+  }) => `
+    background: ${neutral[0]};
+  `}
+  border-radius: 50%;
+  margin: auto;
+  text-align: center;
+
+  ${({
+    theme: {
+      spacing: { medium },
+    },
+  }) => `
+  padding-top: ${medium}px;
+`}
+
+  ${desktopBreakpoint`
+    ${({
+      theme: {
+        spacing: { medium },
+      },
+    }) => `
+    padding-top: ${medium * 1.125}px;
+  `}
+`}
+
+  width: ${MOBILE_OVERLAY_WIDTH}px;
+  height: ${MOBILE_OVERLAY_HEIGHT}px;
+
+  ${desktopBreakpoint`
+    width: ${DESKTOP_OVERLAY_WIDTH}px;
+    height: ${DESKTOP_OVERLAY_HEIGHT}px;
+  `};
+
+`;
+
+const TextWrapper = styled.div`
+  ${({
+    theme: {
+      spacing: { medium },
+    },
+  }) => `
+      margin-left: ${medium}px;
+  `}
+
+  width: 210px;
+  ${desktopBreakpoint`
+    width: 300px;
+  `}
+
+  display: inline-block;
+
+  strong,
+  small {
+    white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+`;
+
+const CurrentText = styled.strong`
+  display: block;
+  height: 26px;
+  ${({ theme: { baseFontSize } }) => `
+    font-size: ${baseFontSize * 1.25}px;
```

Comment:
cannot use `em` instead?

Reply:
We use `em` values in case of 100% responsivity.
In case this case, we follow the style guide thats show only 2 cases of this dynamics, mobile (<1024) and desktop (> 1024).
With this scenario, the layout does not change very much (Only one time). So we prefer use fixed values.

Reply:
Because we use base font size from theme, in pixels.
We could transform to `em` values but, in this case, what would be the benefits?